src/schema/v1/fields/money.ts

```diff
@@ -105,6 +109,37 @@ const money = ({ name, resolve }) => ({
   }),
 })
 
+export const Price = new GraphQLObjectType<any, ResolverContext>({
+  name: "Price",
+  fields: {
+    minor: {
+      type: new GraphQLNonNull(GraphQLInt),
+      description: "An amount of money expressed in minor units (like cents).",
+      resolve: ({ cents }) => cents,
+    },
+    currencyCode: {
+      type: new GraphQLNonNull(GraphQLString),
+      description:
+        "The ISO-4217 alphabetic currency code, as per https://en.wikipedia.org/wiki/ISO_4217",
+      resolve: ({ currency }) => currency,
+    },
+    display: {
+      type: GraphQLString,
+      description: "A pre-formatted price.",
+    },
+    major: {
+      type: new GraphQLNonNull(GraphQLFloat),
+      description:
+        "An amount of money expressed in major units (like dollars).",
+      resolve: ({ cents, currency }) => {
+        const factor = currencyCodes[currency.toLowerCase()].subunit_to_unit
```

Comment:
Unrelated to this PR, looks like we grab the unit <-> subunit conversion from the Ruby money gem and maintain that [ourselves](https://github.com/artsy/metaphysics/blob/ca2f97377170399ff86d3a91c00c284733c66979/src/schema/v1/fields/money.ts#L13-L14). One thing we noticed previously is that it's not rare the mapping changes (and even not matching the ISO standard); in fact, we actually ran into a problem where Gravity and Ohm were using different version of the gem and the subunit to unit ratio was different. We might want to research better ways to maintain the mapping.

Reply:
I think that's a solid recommendation. As it wouldn't be a breaking change, I'd vote we defer this for future action. Maybe it can be brought up in the currency working group?

Reply:
Yes! I simply wanted to raise a previous issue and totally agree we can follow up separately.